src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpClassification.cs

```diff
@@ -106,7 +106,7 @@ static void Main(string[] args)
             VisualStudio.Editor.Verify.CurrentTokenType(tokenType: "class name");
         }
 
-        [WpfFact, Trait(Traits.Feature, Traits.Features.Classification), Trait(Traits.Editor, Traits.Editors.LanguageServerProtocol)]
```

Comment:
The command to change build configuration doesn't seem to be working on the LSP integration test machines. However, I manually verified that things work as expected in an actual scenario.

Reply:
hmm - I think without this trait the LSP tests won't run this test, is that intentional in this PR?  

we should also trigger an LSP run on this PR - should be `/azp run roslyn-integration-lsp-CI`

Reply:
This is intentional because the test is failing in the LSP integration run I kicked off:
https://dev.azure.com/dnceng/public/_build/results?buildId=1760627&view=results

I think it'll warrant further investigation but this seems to be a test issue + not an issue when manually testing